doc/examples/color_exposure/plot_adapt_rgb.py

```diff
@@ -35,11 +35,10 @@ def sobel_each(image):
 def sobel_hsv(image):
     return filters.sobel(image)
 
-"""
-We can use these functions as we would normally use them, but now they work
-with both gray-scale and color images. Let's plot the results with a color
-image:
-"""
+######################################################################
+# We can use these functions as we would normally use them, but now they work
+# with both gray-scale and color images. Let's plot the results with a color
+# image:
```

Comment:
@jni it is necessary indeed (strings within triple quotes are not transformed to text blocks but appear in the code). The rationale between this is that it's better to teach users to put comments with # than within triple quotes (see the discussion in https://github.com/sphinx-gallery/sphinx-gallery/issues/120), since triple quotes should be docstrings.

But it's a bit ugly to read indeed, and it was a pain to implement :-).